public/locales/en/translation.json

```diff
@@ -153,7 +153,7 @@
     "postalCode": "Postal Code",
     "sortingCode": "Sorting code",
     "state": "State / Province",
-    "userRegistrationRequired": "User Registration Required",
+    "Is Public": "Is Public",
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.